src/record/list.js

```diff
@@ -214,7 +226,9 @@ List.prototype._applyUpdate = function( message ) {
 		message.data[ 2 ] = '[]';
```

Comment:
Yeah, I meant if I listen to a record called 'user' which is { name: 'John' } via the list API what it will do is rewrite the content with '[]'

That being said, it's a different issue from this PR and not exactly a priority.